File: backend/src/main/java/com/shorobura/model/Post.java
package com.shorobura.model;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@Entity
public class Post {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String title;
    private LocalDateTime dateTime;
    private String context;

    public Post(int id, String title, LocalDateTime dateTime, String context) {
        this.id = id;
        this.title = title;
        this.dateTime = dateTime;
        this.context = context;
    }

}

File: frontend/src/app/post/post.ts
export class Post {
  private _id: number;
  private _title: string;
  private _dateTime: string;
  private _context: string;


  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get title(): string {
    return this._title;
  }

  set title(value: string) {
    this._title = value;
  }

  get dateTime(): string {
    return this._dateTime;
  }

  set dateTime(value: string) {
    this._dateTime = value;
  }

  get context(): string {
    return this._context;
  }

  set context(value: string) {
    this._context = value;
  }
}
